### ISSUE TITLE
fixups need to run before export as zip or python source

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#2133]
It looks like fixups are no longer run on every simulation when the simulationList is requested and the schema version is changed. In that case, the fixups need to get run on any other path into the data, such as "export as zip" or "python source". Otherwise old data will get used for those methods and could cause a server error.

An example server error - select the SRW example Undulator Radiation, then select Discard Changes to example and close the browser before the getData request occurs. That leaves an un-fixed up simulation. Then from the simulation list, select Export as Zip or Export Python and a server error occurs because the fixup has not been run:

```
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/template/template_common.py", line 371, in validate_model
    raise Exception('no value for field "{}" and no default value in schema'.format(k))
../sirepo/http_reply.py:240:_gen_exception_error unsupported exception=<type 'exceptions.Exception'> msg=no value for field "horizontalDeflectingParameter" and no default value in schema
```

**Comment by robnagler** [comment]
I was able to reproduce the error, but the root cause is something else. The file contains:

```txt
    "sim_data_template_fixup": true,
    "simulationType": "srw",
    "version": "20191217.165908"
```

The problem is that prepare_for_client is not being called before the data is exported. The only code this affects is SRW, because it's the only code that needs to access the code (srwlib) directly in the fixup. 

I think the solution should be to run a program during the fixup for SRW instead of the sim_data_template_fixup hack. sim_data would call, say `sirepo srw fixup foo.json`. this program would read and write the file, running the fixups as necessary. 

We will have py2 in the container. I realize that srw runs py3, but that's besides the point: codes should not run inside the flask server. Treating SRW as a special case, because it is Python, is a mistake.

**Comment by robnagler** [comment]
Fix 8d1ef2356630e30c88ebbc40ea4881ebdccf2309
